server/bin/unittests

```diff
@@ -59,6 +59,19 @@ function _run_indexing {
     _run pbench-index --tool-data
 }
 
+function _run_pytest {
+    tname=${1}
+    export PYTHONPATH=${_testopt}/lib:$PYTHONPATH
+    shift
+    if [[ -z "${@}" ]]; then
+        echo "+++ Running ${tname}" >> ${_testout}
+    else
+        echo "+++ Running ${tname} ${@}" >> ${_testout}
+    fi
+    pytest -q ${_testopt}/pytests/${tname}.py
```

Comment:
This needs to be redirected to the _testout file, but configparser needs to be modifed so that the SafeConfigParser deprecation warnings disappear.

Reply:
If you add the following line in `_run_pytest` then the deprecation warning is eliminated, so we can redirect the output and not worry about them:

    export PYTHONWARNINGS="ignore::DeprecationWarning:configparser" 

That's a temporary step that allows you to get the "correct" output in the gold files (i.e. without the deprecation warnings), but still allows reviewers to run the unit tests without having to modify anything on their side.

In the longer term, I'll make the change in configtools and release an updated RPM (bumping up the version in the process, so that when people upgrade pbench-agent or pbench-server, they will also update their configtools installation). At some point in the future, after everybody has upgraded, we can then get rid of the `PYTHONWARNINGS` export in a subsequent PR.

Reply:
sure